packages/patternfly-4/react-core/src/components/BackgroundImage/BackgroundImage.js

```diff
@@ -1,9 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
 import { css } from '@patternfly/react-styles';
+import { css as emotionCSS, cx } from 'emotion';
```

Comment:
This can be done with everything exported from react-styles
```jsx
import { StyleSheet, css } from '@patternfly/react-styles'

const styles = StyleSheet.create({
  bgOverrides: `&.pf-c-background-image { ...overrides }`
});
<div {...props} className={css(styles.backgroundImage, styles.bgOverrides, className)}>
```

This will avoid adding emotion as a direct dep and have it pulled in through react-styles. The only reason the syntax of `StyleSheet.create` was kept vs using `cx/css` was to prevent from having a breaking change. If koku is up for a large overhaul it would be a good change to use cs/css instead.

Reply:
Unfortunately, `StyleSheet.create` did not work with all the scenarios tested. It worked fine with the react-docs example, but did not work when using the component outside of that environment -- the styles were not injected into the page. 

This does work fine when used in Koku login page, for example -- just not when the code is used by the component. If I invoke emotion wityhout going through the create step, it works as expected. I suspect there was an issue with `StyleSheet.create` and pushing `filteredStyles` to emotion.cx. Regardless, it doesn't sound like we should be doing this inside the component.